cli/remotebazel/remotebazel.go

```diff
@@ -537,6 +539,25 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 		envVars[envVar] = val
 	}
 
+	guid, err := uuid.NewRandom()
+	if err != nil {
+		return 0, err
+	}
+	invocationID := guid.String()
+
+	// On SIGTERM, cancel the remote run
```

Comment:
just as fyi, pressing `Ctrl+C` sends SIGINT (`== os.Interrupt` on posix). wasn't sure if you already knew that since the code is mostly referencing SIGTERM which is less common :)

Reply:
Oops no I did not - changed some variable names